README.md

```diff
@@ -1,47 +1,84 @@
+
+Use npm to install and update project dependencies:
 
 ```sh
 npm install
 ```
 
-To create a production build:
+### Setup for local development
+
+To run the project locally, you need to follow two additional steps:
 
+1. install version 16 of [now CLI](https://zeit.co/download) globally:
 ```sh
-npm run build
+npm install -g now@16
 ```
+2. [link project to Google Calendar API](#set-up-google-calendar-API) (see instructions below) 
 
-To run in development
+### Set up Google Calendar API
 
-Install Now CLI
+To link the project to a Google Calendar account, **add a Google API Key** to a newly created **environment file**:
 
-```sh
-npm install -g now
+1. **get API key and id** for a Google calendar service
+   - to create or access this content, check [Google's "Creating a Google API Key" tutorial](https://docs.simplecalendar.io/google-api-key/)
+2. create an `.env` file in the root folder and insert:
+```env
+GOOGLE_CALENDAR_ID=<calendar_id>
+GOOGLE_CALENDAR_API_KEY=<api_key>
+API_HOST=http://localhost:3000
 ```
+3. in the `.env` file, replace:
+   - <calendar_id> with your Google Calendar's ID
+   - <api_key> with its matching API key
 
-To test with your own google calendar, you need to follow these steps:
-1) Register a project on https://code.google.com/apis/console
-2) Generate a Simple API Access key
-3) Ensure Calendar API is activated under services.
-4) Your test calendar must be public (on calendar settings)
+## Make a production build
+
+After [setting up the project](#project-setup), run:
 
-On a .env file, add the following keys from Google Calendar API
 ```sh
-GOOGLE_CALENDAR_ID=<id_of_your_calendar_available_on_calendar_settings>
-GOOGLE_CALENDAR_API_KEY=<simple_key_generated_on_google_projects_console>
-API_HOST=<the_address_where_the_api_is_running_by_default_http://localhost:3000>
+npm run build
 ```
 
-Next, run now in development mode
+## Run the project locally
+
+After [setting up the project](#project-setup) (including [steps for local setup](#set-up-for-local-development)), run:
+
 ```sh
 now dev
 ```
 
-Open a browser on `localhost:3000`
+You can **access the app** in `localhost:3000`.
+Serverless functions are also available in `localhost:3000/api/<serverlessfunction>`, by replacing <serverlessfunction> with the desired function you want to run.
+
+## Contribute to the project
+
+First of all, we thank you for helping us with this project!
+
+To contribute to [geeksessions.io], please follow the steps below:
+
+1. either [**choose an open issue** with no assignee](https://github.com/geeksessions/geeksessions.io/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee), or [create an issue](https://github.com/geeksessions/geeksessions.io/issues/new/choose) for a new feature
+2. **fork the repository**
+   - check [Github's "Forking Projects" guide](https://guides.github.com/activities/forking/#fork) for additional support
```

Comment:
`for additional support` should be made clearer. Maybe `if you need help with this step`